### PR TITLE
User registration

### PR DESCRIPTION
**Pull request by kmbradshaw**

### CODE REVIEW

File: db/migrate/20160924010603_create_users.rb [hunk 1]
```diff
@@ -6,6 +6,7 @@ def change
       t.string :last_name
       t.string :email
       t.string :password_digest
+      t.string :status, default: "pending"
```
Comment (laurieskelly) [inline db/migrate/20160924010603_create_users.rb]: @kmbradshaw don't forget to remove this. are you going to add the new column migration or did I say I would do that?

Comment (kmbradshaw) [reply db/migrate/20160924010603_create_users.rb]: I got it! just wanted to finish up that user settings change first, which I just did :)

Comment (kmbradshaw) [reply db/migrate/20160924010603_create_users.rb]: @laurieskelly @benedictify @pauledwardmitchell this can be merged now!

**Comment by benedictSchurwanz** [comment]
per Kaitlyn, "expiration" code from users_controller needs to be added back in, was taken out because the migration for that isn't there yet, will be back in next update

**Pull request merged**